client/pages/sections/granted/protocol-establishments.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
```

Comment:
As above, this did have some behaviour changes initially but they were reverted - decided to keep the refactor to using hooks.